src/Microsoft.OData.Core/Json/ODataUtf8JsonWriter.TextWriter.cs

```diff
@@ -142,8 +142,18 @@ private bool CheckIfWritingJson(string currentContentType)
         internal sealed class ODataUtf8JsonTextWriter : TextWriter
         {
             private readonly ODataUtf8JsonWriter jsonWriter = null;
+            // Buffer used to store chars that could not be encoded due to
+
+            // insufficient data in the input buffer. The chars will be prepended
+            // to the next chunk of input.
             private char[] buffer;
+            // Number of chars in the buffer that are left over from the previous chunk.
+
             private int numOfCharsNotWrittenFromPreviousChunk = 0;
+            // This buffer is used to Write(char) to store the char so
+
+            // that we can re-use our Write(char[], ...) method.

```

Comment:
nit: Remove empty line in the middle of the statement
```suggestion
            // This buffer is used to Write(char) to store the char so
            // that we can re-use our Write(char[], ...) method.
```